jetty-core/jetty-util/src/test/java/org/eclipse/jetty/util/thread/ReservedThreadExecutorTest.java

```diff
@@ -174,12 +171,12 @@ public void testEvict() throws Exception
         _executor.startThread();
         _executor.startThread();
 
-        waitForAvailable(2);
+        waitAtMost(10, SECONDS).until(_reservedExecutor::getAvailable, is(2));
 
         int available = _reservedExecutor.getAvailable();
         assertThat(available, is(2));
-        Thread.sleep((5 * IDLE) / 2);
-        assertThat(_reservedExecutor.getAvailable(), is(0));
+
+        waitAtMost(5 * IDLE, MILLISECONDS).until(_reservedExecutor::getAvailable, is(0));
```

Comment:
You're now waiting up to twice as long, 5 times IDLE instead of `(5 * IDLE) / 2`. Is that on purpose?

Reply:
Yep, that shorter wait was always done, hence kept short.   Now we only wait until we see the condition.